data/spec-lanes.json

```diff
@@ -44,6 +55,90 @@
                         "description": "Traffic does not travel in this lane."
                     }
                 ]
+            },
+            "designated": {
+                "type": "string",
+                "description": "The predominant or designated traffic type, to hint at how to depict the lane. Should not be used to determine access restrictions.",
```

Comment:
Should not be used for access restrictions -- why not? How else do we know that a cycle lane alongside a road doesn't allow cars to squeeze in and drive through? Aside from this second sentence, this look good to me

Reply:
I would prefer to have another key: `access`, which contains a tags matching https://wiki.openstreetmap.org/wiki/Key:access#Transport_mode_restrictions for that specific lane.

A counterexample for designated=bicyclem, but cars allowed, is https://nl.wikipedia.org/wiki/Fietsstraat

Reply:
Got it. From a routing perspective, this sounds like the difference between "allowed" and "encouraged" -- driving down a "living street" is acceptable, but not prioritized. Cycling on a path through a park primarily meant for foot traffic is allowed, but not expected to be fast.